src/wp-includes/block-template-utils.php

```diff
@@ -155,7 +155,11 @@ function get_default_block_template_types() {
 		),
 		'date'           => array(
 			'title'       => _x( 'Date', 'Template name' ),
-			'description' => __( 'Displays a post archive when a specific date is visited (e.g., example.com/2023/).' ),
+			/* translators: %s: Example URL. */
+			'description' => sprintf(
+				__( 'Displays a post archive when a specific date is visited (e.g. %s).' ),
+				'<code>example.com/2023/</code>'
```

Comment:
@SergeyBiryukov The `<code>` tags will be removed as they do not render in React. But question for you? Should the URL be directly in the `__()` or as it is in a `%s` placeholder?

Reply:
As the `example.com/2023/` part should likely be translatable, I think it's fine to include directly in the string.

There was a similar instance in [r55452](https://core.trac.wordpress.org/changeset/55452), also directly in the string.

Reply:
Awesome, thank you @SergeyBiryukov. I'll revert the commit that changed this string to put the URL back into the translatable string.

Reply:
Oh wait, I see looking at r55452. Put the URL into it's own translatable string. Aha. Will do 👍